### PR TITLE
Text Reflowing Justification (3)

### PR DESCRIPTION
**Pull request by LincolnAuster**
See #236

### CODE REVIEW

**Review by jmacdonald — COMMENTED** [review]
Just a few small suggestions. Once those are in place, I think we'll be all set to merge. 😎

File: src/commands/selection.rs [hunk 1]
```diff
@@ -100,6 +80,49 @@ fn copy_to_clipboard(app: &mut Application) -> Result {
...
+	},
+	Mode::Search(ref search) => {
+	    Ok(search
+		.results
+		.as_ref()
+		.and_then(|r| r.selection())
+		.ok_or("A selection is required.")?
+		.clone())
+	}
+	_ => bail!("A selection is required."),
```
[the remark this answers is not in the document]
Comment (LincolnAuster) [reply src/commands/selection.rs]: Ah, thanks for catching that! Fixed with 3d60d28!

File: src/commands/selection.rs [hunk 2]
```diff
@@ -100,6 +80,49 @@ fn copy_to_clipboard(app: &mut Application) -> Result {
...
+
+    match app.mode {
+	Mode::Select(ref sel) => {
+	    let cursor_position = *buf.cursor.clone();
+	    Ok(Range::new(cursor_position, sel.anchor))
+	},
+	Mode::SelectLine(ref sel) => {
+	    Ok(util::inclusive_range(
+    		&LineRange::new(
+    		    sel.anchor,
```
[the remark this answers is not in the document]
Comment (LincolnAuster) [reply src/commands/selection.rs]: Thanks- fixed in 3a640bd!

File: src/input/key_map/default.yml [hunk 3]
```diff
@@ -218,6 +218,7 @@ select:
   R: git::copy_remote_url
   m: view::scroll_down
   f: application::switch_to_second_stage_jump_mode
+  z: selection::justify
```
[the remark this answers is not in the document]
Comment (LincolnAuster) [reply src/input/key_map/default.yml]: adf2a8d moves justify to `Z`, and adds suspend add `z` in the requisite modes, as that seems to be the desired behavior --- is that correct?

[the remark this answers is not in the document]
Comment (jmacdonald) [reply src/input/key_map/default.yml]: Yes, thanks for addressing that while you were in there!

**Pull request merged**